api/proto/config/storage.proto

```diff
@@ -242,3 +243,48 @@ message DisksConfig {
   DisksArrayType array_type = 2;
   repeated DisksConfig children = 3;
 }
+
+// SnapshotOpsCmd counter for snapshot cmd
+message SnapshotOpsCmd {
+  uint32 counter = 1;
+  google.protobuf.Timestamp opsTime = 2;
+}
+
+// ZfsSnapshot describes a snapshot for a specific logical
+// volume that must exist on the device. It has a required
+// field volume_uuid for which it was created.
+message ZfsSnapshot {
```

Comment:
Should we name it with zfs in the beginning? Probably it is better to have "zfs" in some kind of "type" field? And the same question for info.

Reply:
Why not? There is no guarantee that we will have a different type of snapshot, and that the other type will have the same fields as the snapshot from zfs.

Reply:
Renamed